pkg/operator/ceph/cluster/controller.go

```diff
@@ -739,13 +847,41 @@ func (c *ClusterController) handleDelete(cluster *cephv1.CephCluster, retryInter
 	return nil
 }
 
+func purgeExternalCluster(clientset kubernetes.Interface, namespace string) error {
```

Comment:
These resources to purge were all created by the import script? Ideally we would add the OwnerReferences to them to their cluster CR parent, then K8s will clean them up automatically.

Reply:
Sounds good but what if they get created differently?